pep-0596.rst

```diff
@@ -0,0 +1,191 @@
+PEP: 596
+Title: Use UTF-8 for default text file encoding
+Author: Inada Naoki  <songofacandy@gmail.com>
+Status: Draft
+Type: Standards Track
+Content-Type: text/x-rst
+Created: 04-Jun-2019
+Python-Version: 3.9
+
+Abstract
+========
+
+Currently, ``TextIOWrapper`` uses ``locale.getpreferredencoding(False)``
+when ``encoding`` is not specified.
+
+This PEP proposes changing the default encoding to "UTF-8" regardless of
+platform or locale.
```

Comment:
I made a similar proposition in 2011, but it was too early :-)
"[Python-Dev] open(): set the default encoding to 'utf-8' in Python 3.3?"
https://mail.python.org/pipermail/python-dev/2011-June/112086.html